simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -367,6 +370,43 @@ public void live_reloading() {
                 .run();
     }
 
+    @Test
+    public void uncommon_flakes() {
+        servers = servers(
+                SimulationServer.builder()
+                        .serverName("fast0")
+                        .simulation(simulation)
+                        .handler(h -> h.response(respond500Every(100)).responseTime(Duration.ofNanos(1000)))
+                        .build(),
+                SimulationServer.builder()
+                        .serverName("fast1")
+                        .simulation(simulation)
+                        .handler(h -> h.response(respond500Every(100)).responseTime(Duration.ofNanos(1000)))
```

Comment:
i've found anything that is exactly regular like this can have weird effects, e.g. if you have two nodes and pure round robin, then all of these 500s will exactly land on one of the nodes. Would recommend using a pseudorandom instance to put a bit of jitter into these (e.g. perhaps varying between 90 and 110?)

Reply:
:+1: